README.md

```diff
@@ -88,3 +88,22 @@ drop into the shell.
 ### Load addresses:
 
 `./scripts/load-addresses` (from `out/addresses.json` to ENV variables)
+
+## Nix
+
+To be able to easily share scripts with other repos and make sure that the
+exact dependencies needed to run the scripts are met we can use Nix.
+
+After changing submodules the lock file (`nix/dapp.nix`) needs to be updated
+using:
+
+```sh
+nix run -f https://github.com/icetan/dapp2nix/tarball/master -c dapp2nix > nix/dapp.nix
```

Comment:
Maybe we should instead add `dapp2nix` to `shell.nix`? Seems easier for users to just have to type

```
$ nix-shell
$ dapp2nix > nix/dapp.nix
```

Reply:
True, maybe examples of both should be in  `README.md` as you might not want to have to run the shell hooks in `shell.nix` just to update the lock file.

Could change the `nix run` to `nix-env` instead so that you have `dapp2nix` in your path, but that might not be desirable for some.

Reply:
What's the problem with running the shell hooks? Do they take long?

I would probably optimize for the simplest case at first (adding it to the `shell.nix`), and more advanced users will know how to use `nix run` anyway.

Reply:
Will remove `nix run` block.